packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -35,12 +37,17 @@ function DropZone(props: SpectrumDropZoneProps, ref: DOMRef<HTMLDivElement>) {
   let domRef = useDOMRef(ref);
   let messageId = useId();
   let stringFormatter = useLocalizedStringFormatter(intlMessages);
+  let isVirtualDrag = isVirtualDragging(); 
```

Comment:
Moved the `isVirtualDrag` to RSP DropZone so that we only pass the `aria-labelledby` of the filled message when the dropzone is in both a filled and virtual drag state. 

The reason for including `useDragSession` is because most of the re-rendering is happening in RAC DropZone. So when you call `isVirtualDragging` by itself in RSP DropZone, the value isn’t getting updated. `useDragSession` basically allows us to subscribe to when `beginDragging` is called so the a re-render in RSP DropZone can happen and `isVirtualDragging`’s value can be updated.